coding-concepts.js

```diff
@@ -10,37 +10,38 @@
 var cohort = "Echo 2021"
 // console.log(cohort.split(""))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer:  The log will convert the string into an array using the split method. ['E', 'c', 'h', 'o', ' ', '2', '0', '2', '1']
+// b) Verify and explain:  The accessor method of, .split(""), conveted the string "Echo 2021", into an array of individual string characters. 
 
 
 // --------------------2) What will this log?
 
-const greeter = (name) => {
+function greeter(name) {
   `Hello, ${name}!`
 }
 // console.log(greeter("LEARN Student"))
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: This log will more than likely yield a result of undefined as the string "LEARN Student"has not been delcared in this code, also it does not have return.
+
+//(b) Verify and explain:  The code ran undefined as indicated in my answer. The functions does not meet the criterias of a function. The code is missing, the single equal sign, the arrow function syntax, and the names are not camelCased.  In retrospect, I took this code more as a function but, I know it is an string interpolation, please advise :) 
```

Comment:
You don't need to change anything about the code itself. The question is asking what you expect to see as an output.
And you are correct, it is undefined. So that is exactly the challenge. Why is it undefined?